src/sagas/helpers.js

```diff
@@ -138,3 +139,30 @@ export function isUnlockScreen(action) {
   return action.type === types.SET_LOCK_SCREEN
     && action.payload === false;
 }
+
+/**
+ * Get registered tokens from the wallet instance.
+ * @param {HathorWallet} wallet
+ * @param {boolean} excludeHTR If we should exclude the HTR token.
+ * @returns {string[]}
+ */
+export async function getRegisteredTokens(wallet, excludeHTR = false) {
```

Comment:
Redux-saga does not support iterating on async generators (see [ref](https://github.com/redux-saga/redux-saga/blob/8f60bcb27b088b4785bbd106eaf6cded13d99562/packages/core/src/internal/proc.js#L13)) so we cannot use the `for await` syntax to get the registered tokens from the storage.

The solution is to manually iterate on the asnc generator, this was encased in this helper.
If we need to create more helpers like this we can create a more generic `asyncGeneratorToArray` with the while logic.